units/unit.cpp

```diff
@@ -23,7 +23,8 @@ Unit::Unit(const std::string &unit)
 std::string Unit::name() const {
   auto repr = to_string(m_unit);
   repr = std::regex_replace(repr, std::regex("^u"), "µ");
-  repr = std::regex_replace(repr, std::regex("item"), "counts");
+  repr = std::regex_replace(repr, std::regex("item"), "count");
```

Comment:
Why singular now?

Reply:
So I don't need to check for `items`. This will replace the `item` part, and if it was singular the next line will change to plural.